github/files/vagrant/puppet/puphpet/ansible/php.yml

```diff
@@ -29,6 +29,7 @@
     php_upload_max_filesize: 64M
 
     php_enable_apc: true
+    php_apc_enabled_in_ini: true
```

Comment:
I'll wait for final diff one, because currently it has this mod https://github.com/propeoplemd/cibox/pull/197/files